src/shared/bg_pmove.cpp

```diff
@@ -4667,14 +4667,7 @@ static void PM_HumanStaminaEffects()
 	}
 
 	// Check stamina limits
-	if ( stats[ STAT_STAMINA ] > STAMINA_MAX )
-	{
-		stats[ STAT_STAMINA ] = STAMINA_MAX;
-	}
-	else if ( stats[ STAT_STAMINA ] < 0 )
-	{
-		stats[ STAT_STAMINA ] = 0;
-	}
+	Math::Clamp( stats[ STAT_STAMINA ], 0, STAMINA_MAX );
```

Comment:
This doesn't do anything. Use this instead:
```suggestion
	stats[ STAT_STAMINA ] = Math::Clamp( stats[ STAT_STAMINA ], 0, STAMINA_MAX );
```